index.html

```diff
@@ -1592,17 +1590,39 @@ <h3 id="applying">
         <div class="issue" data-number="446"></div>
       </section>
     </section>
-    <section>
+    <section data-dfn-for="WebAppManifest" data-link-for="WebAppManifest">
       <h2>
         Manifest and its members
       </h2>
       <p>
         A <dfn>manifest</dfn> is a JSON document that contains startup
         parameters and application defaults for when a web application is
-        launched. A manifest consists of a top-level <a>object</a> that
-        contains zero or more members. Each of the members are defined below,
-        as well as how their values are processed.
+        launched. A manifest consists of a top-level <dfn>WebAppManifest</dfn>
+        <a>dictionary</a> that contains zero or more members.
+        Each of the members are defined below, as well as how their values are processed.
       </p>
+      <pre class="idl">
```

Comment:
Nit: I prefer seeing the IDL before the description... "an IDL block speaks a thousand words" 😂